Source/utils/utf8.hpp

```diff
@@ -23,7 +23,7 @@ char32_t DecodeFirstUtf8CodePoint(string_view input, uint8_t *len);
  */
 inline char32_t ConsumeFirstUtf8CodePoint(string_view *input)
 {
-	uint8_t len;
+	uint8_t len = 0;

```

Comment:
`len` is now always set by `DecodeFirstUtf8CodePoint`, so perhaps the initialization here isn't needed?

Reply:
Yeah, clang wasn't warning about this before presumably because the function took a pointer. This was a first attempt while I was working out what the actual cause of the failure was. Of course now clang isn't warning about unused intialisation value... So not sure whether it's safer to leave it or remove it.

Reply:
Well, we shouldn't initialize it unless necessary, and it doesn't seem necessary!